### ISSUE TITLE
DNS for various domain aliases is broken

### ISSUE THREAD

**Issue by lilyball** [rust-lang/rust-www#21]
We appear to have a few domain aliases for www.rust-lang.org other than rust-lang.org. The ones I'm aware of are rust-lang.com, rust-lang.net, rustlang.com, rustlang.org, and rustlang.net. There may be others too, I haven't gone looking.

The DNS for some of these is ok and for others it's broken. "Broken" means that visiting the page takes you to the easydns.com parked domain page.

Via `dig`, it seems that the correct domains all have an A record for 72.8.141.90, which is the IP for fwd.st2.easydns.com. These correct domains are rust-lang.net, rustlang.com, rustlang.net, and of course rust-lang.org. These all get redirected to www.rust-lang.org.

Unfortunately, others have 216.220.40.248, which is the parked domain DNS. The bad domain here is rust-lang.com.

You'll note I haven't mentioned rustlang.org yet. That's because it actually has _both_ A records. This means that a visitor may arbitrarily either get redirected correctly, or may see the parked domain page. I personally can get both, by using different browsers.

So in short, the DNS for the following domains needs to be fixed to use the correct A address: rustlang.org, rust-lang.com.
